api/desecapi/models.py

```diff
@@ -227,3 +230,68 @@ def save(self, *args, **kwargs):
 
     class Meta:
         ordering = ('created',)
+
+
+class RRset(models.Model):
+    # TODO Do these two fields really make sense? Meaning is limited when deleting + recreating an RRset
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(null=True)
+    domain = models.ForeignKey(Domain, on_delete=models.CASCADE, related_name='rrsets')
+    subname = models.CharField(max_length=178, blank=True)
+    type = models.CharField(max_length=10)
+    records = models.CharField(max_length=64000, blank=True)
+    ttl = models.PositiveIntegerField(validators=[MinValueValidator(1)])
+    _dirty = False
+
+    class Meta:
+        unique_together = (("domain","subname","type"),)
+
+    def __setattr__(self, attrname, val):
+        setter_func = 'setter_' + attrname
+        if attrname in self.__dict__ and callable(getattr(self, setter_func, None)):
+            super().__setattr__(attrname, getattr(self, setter_func)(val))
+        else:
+            super().__setattr__(attrname, val)
+
+    def setter_records(self, val):
+        if val != self.records:
+            self._dirty = True
+
+        return val
+
+    def setter_ttl(self, val):
+        if val != self.ttl:
+            self._dirty = True
+
+        return val
+
+    def update_pdns(self):
+        from .serializers import RRsetSerializer
```

Comment:
Because `serializers.py` globally imports `Domain` which is in `models.py`. If `models.py` now imports something from `serializers.py` globally, there will be an `import` loop going back and forth when loading either file.

I know this is ugly, but I don't know how to solve, except by breaking the loop by a local import. Any other ideas?